docs/tutorial/notebook/02_datasets.ipynb

```diff
@@ -0,0 +1,449 @@
+{
```

Comment:
<p>Remove Open3D-ML for consistency</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>